src/drivers/qnx/lv_qnx.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file lv_qnx_window.h
+ *
+ */
+
+#ifndef LV_QNX_DISP_H
+#define LV_QNX_DISP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "../../display/lv_display.h"
+#include "../../indev/lv_indev.h"
+
+#if LV_USE_QNX
+
+#include <stdbool.h>
+#include <screen/screen.h>
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+lv_display_t * lv_qnx_window_create(int32_t hor_res, int32_t ver_res);
```

Comment:
Please add API comments here.

Reply:
* Added the OS readme file. Once we get further ahead I'll probably add a recipe for building LVGL as either a shared or static library, instead of having to add it to every project separately.
* Added Doxygen comments for the API
* I did not enable any of the monitors, so I'm still not clear as to why a wakeup is needed every 33ms
* Needs more input drivers for keyboard and touchscreen (at least)